src/object_store/src/object/opendal_engine/oss.rs

```diff
@@ -33,8 +27,6 @@ impl OpendalObjectStore {
 
         builder.bucket(&bucket);
 
-        builder.write_min_size(OSS_PART_SIZE);
```

Comment:
Should we adopt the buffer option to create the writer given that `write_min_size` is deprecated? https://opendal.apache.org/blog/#buffered-writer

Reply:
nvm. I just noticed that we already use the buffer option.

Reply:
Yes, already use it when last time bumping version.